GiniVision/Classes/Core/CameraViewController.swift

```diff
@@ -228,6 +248,16 @@ public typealias CameraScreenFailureBlock = (_ error: GiniVisionError) -> Void
         NotificationCenter.default.removeObserver(self)
     }
     
+    fileprivate func didPick(validatedDocuments documents: [GiniVisionDocument]) {
+        if let delegate = delegate {
+            delegate.camera(self, didCaptureDocuments: documents)
+        } else if let firstDocument = documents.first {
+            successBlock?(firstDocument)
+        } else {
+            print("It has not been specified a CameraViewControllerDelegate")
```

Comment:
"A CameraViewControllerDelegate has not been specified".

Also, isn't it time to have a better logging solution? It's a library after all.

Reply:
I thought about it when adding TrustKit to the other SDK but I forgot to add it to Github. Thanks for reminding me about it!